src/uuid/validator/uuidvalidator.rs

```diff
@@ -77,7 +69,7 @@ pub trait UuidValidator {
     /// # Errors
     ///
     /// Returns a `ValidationError` with "Invalid UUID Time" if the UUID's time component is invalid or not properly formatted.
-    fn validate_time(&self, uuid: &uproto_Uuid) -> Result<(), ValidationError> {
+    pub fn validate_time(uuid: &uproto_Uuid) -> Result<(), ValidationError> {
         if let Some(time) = UuidUtils::get_time(uuid) {
```

Comment:
I suppose a side-effect of my suggestion [here](https://github.com/eclipse-uprotocol/uprotocol-rust/pull/9/files/092aea461fde0386e43be3c869e3065e7b2cbc7d#r1429467976) about removing support for UUIDv6 from `get_time()` would mean this now fails to validate.

I suppose the v6 UUID would fail to validate on the call to `validate_version()` in any case though.

Reply:
yes